manifests/third_party/resourcegroup-manifest.yaml

```diff
@@ -357,6 +357,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
```

Comment:
Remove this (cause of presubmit failure)

Reply:
Yup, I'm just waiting on the e2e statuses first before pushing a new update.

Reply:
Done. Prior e2e tests also passed fyi.